var/spack/repos/builtin/packages/hpx/package.py

```diff
@@ -15,6 +15,8 @@ class Hpx(CMakePackage, CudaPackage):
     maintainers = ['msimberg', 'albestro']
 
     version('master', git='https://github.com/STEllAR-GROUP/hpx.git', branch='master')
+    version('stable', git='https://github.com/STEllAR-GROUP/hpx.git', tag='stable')
+    version('1.4.1', git='https://github.com/STEllAR-GROUP/hpx.git', tag='1.4.1')
```

Comment:
For numbered versions a `sha256` pointing to a tarball is preferable to a git tag.